server/server.js

```diff
@@ -213,10 +213,10 @@ app.delete('/definitions/:id/likes', checkJwt, function (req, res) {
 });
 
 
-// GET users likes/submissions
+// GET users likes
 app.get('/users/:id/likes', checkJwt, function (req, res) {
   acronymsModel.findAcronyms({
-    $or: [{ 'definitions.likes': req.params.id }, { owner: req.params.id }]
+    $or: [{ 'definitions.likes': req.params.id }]
```

Comment:
Since we are removing the second condition, does this still need to be an `$or`?